admin/schema.php

```diff
@@ -513,7 +515,7 @@
 	project_id				I		UNSIGNED NOTNULL DEFAULT '0',
 	user_id					I		UNSIGNED NOTNULL DEFAULT '0',
 	name					C(128)	NOTNULL DEFAULT \" '' \",
-	status					I		UNSIGNED NOTNULL DEFAULT '0' ",
+	status					I		UNSIGNED NOTNULL DEFAULT '1' ",
```

Comment:
Same comment as for *api_token* table below, bug for schema step 212.

Reply:
Comment added.